dev/test/bundle.ts

```diff
@@ -158,7 +168,14 @@ describe('Bundle Buidler', () => {
     );
 
     // Verify named query
-    const namedQuery = (elements[1] as IBundleElement).namedQuery;
+    let namedQuery = (elements[1] as IBundleElement).namedQuery;
+    let newNamedQuery = (elements[2] as IBundleElement).namedQuery;
+    if (namedQuery?.name === 'test-query-new') {
+      // Swap
+      const q = namedQuery;
+      namedQuery = newNamedQuery;
+      newNamedQuery = q;
+    }
```

Comment:
How about this:

```suggestion
    elements.sort((l, r) => l.nameQuery.name < r.nameQuery.name));
    let namedQuery = (elements[1] as IBundleElement).namedQuery;
    let newNamedQuery = (elements[2] as IBundleElement).namedQuery;
```

Or this:

```suggestion
    let namedQuery = elements.find(l.nameQuery.name === 'foo')!;
    let newNamedQuery = elements.find(l.nameQuery.name === 'bar')!;
```